whitelabelco/sofo-foods
# Cp/feat/update careers page

### Description 💻

Resolves: #172 & #186 

### Implementation 💡

- updates copy on Careers page
- redesign Careers page: [Comp](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=322-5865&node-type=frame&t=902Ld4UQFQJbXKht-0)
- create Jobs page for listing: [Comp](https://www.figma.com/design/04vc8B8NFKlryNS0e1h57T/SoFo-Foods?node-id=1242-6255&node-type=frame&t=902Ld4UQFQJbXKht-0)
- implements script provided by client to list open positions


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click on Careers
- match to comps and copy in ticket
- click both buttons on Careers page and confirm is navigates to `/jobs`
- compare jobs to comps
- confirm script loads jobs listing widget

### Screenshots/Demos 📷

### Checklist ✅
